docusaurus.config.js

```diff
@@ -150,6 +130,18 @@ const config = {
                 label: 'Twitter',
                 href: 'https://twitter.com/openfeature',
               },
+              {
+                label: 'LinkedIn',
+                href: 'https://www.linkedin.com/company/openfeature/',
+              },
+              {
+                label: 'Website',
+                href: 'https://openfeature.dev/'
+              },
+              {
+                label: 'Community',
```

Comment:
Perhaps this should say something like "Join the Community". It feels weird to have a community item under the community column.